larray/core/session.py

```diff
@@ -490,9 +490,9 @@ def to_csv(self, fname, names=None, display=False, **kwargs):
         """
         self.save(fname, names, ext_default_engine['csv'], display=display, **kwargs)
 
+    @deprecated(to_csv)
```

Comment:
I don't really like having those old method being defined with no body. I would rather write the "decorator" such that this works:
 
```python
dump_excel = deprecated(to_excel)
```

possibly renaming deprecated to something like renamed_to, old_name_for or deprecated_name_for.

OR

```python
@deprecated('dump_excel')
def to_excel(self, fname, names=None, *args, **kwargs):
    self.save(fname, names, ext_default_engine['xlsx'], overwrite, display, **kwargs)
```
possibly renaming deprecated to something like renamed_from, deprecated_name_was, ...